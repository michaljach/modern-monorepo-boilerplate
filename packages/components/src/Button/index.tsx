import React, { ReactElement } from 'react';

const Button = (): ReactElement => (
  <button>Hello from shared button</button>
);

export default Button;