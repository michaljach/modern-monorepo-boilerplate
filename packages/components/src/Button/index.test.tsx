import React from 'react';
import Button from '.';
import renderer from 'react-test-renderer';

it('renders correctly', (): void => {
  const tree = renderer
    .create(<Button />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});