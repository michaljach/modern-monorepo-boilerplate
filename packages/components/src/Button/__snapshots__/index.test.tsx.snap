// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button>
  Hello from shared button
</button>
`;
